src/libraries/System.Console/tests/ManualTests/ManualTests.cs

```diff
@@ -96,18 +117,6 @@ static string RenderKeyChord(ConsoleKeyInfo key)
             }
         }
 
```

Comment:
@eiriktsarpalis this is a surprise. This test verifies a `ReadLine` against `OpenStandardInput` doesn't echo. Can you check whether the following code echos or not on Windows?
```cs
            var stream = Console.OpenStandardInput();
            var textReader = new System.IO.StreamReader(stream);
            var result = textReader.ReadLine();
```

Reply:
@eiriktsarpalis while your looking at Console tests, can you check if `OpenStandardInput` test does/doesn't echo on Windows?

Reply:
You're right, I just ran the test on Windows and it _does_ in fact echo the characters.

Reply:
Can you update the test description and perhaps address the echoing behaviour on Linux in this PR?